store.go

```diff
@@ -33,8 +33,9 @@ var commonDocRoots = []string{
 	"/var/www/*",
 	"/var/www/*/public_html",
 	"/var/www/vhosts/*/htdocs",
-	"/vhosts/*/httpdocs", // plesk
-	"$HOME/*.*",          // generic domain pattern
+	"/vhosts/*/httpdocs",   // plesk
+	"$HOME/public_html/..", // nexcess
+	"$HOME/??*.*",          // generic domain pattern
```

Comment:
I'm not sure what the double `?` is, but I assume you know what you're doing here :)

Reply:
minimum two characters?

Reply:
So I assume that's a glob thing? :)